docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2038,7 +2042,7 @@ Setting `0` means the short links are cleaned up approximately every 10 minutes.
 A negative value such as `-1` disables expiry.
 
 {{< admonition type="caution" >}}
-Short links without an expiration increase the size of the database and can't be deleted.
+Short links without an expiration increase the size of the database and can't be deleted. Grafana recommends using a duration based on your specific use-case
```

Comment:
Just changed "using" to "setting" to avoid repetition.

```suggestion
Short links without an expiration increase the size of the database and can't be deleted. Grafana recommends setting a duration based on your specific use case
```